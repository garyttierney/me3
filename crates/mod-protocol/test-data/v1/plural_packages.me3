profileVersion = "v1"

[[packages]]
id = "test-pkg"
path = "."
