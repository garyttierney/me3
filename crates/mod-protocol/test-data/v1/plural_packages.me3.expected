V1(
    ModProfileV1 {
        supports: [],
        natives: [],
        packages: [
            Package {
                inner: ModFile {
                    name: "test-pkg",
                    path: ".",
                    enabled: true,
                    optional: false,
                },
                load_before: [],
                load_after: [],
            },
        ],
        savefile: None,
        start_online: None,
        disable_arxan: None,
    },
)
